src/components/header/_macro-options.md

```diff
@@ -115,3 +117,14 @@
 | name       | string | false    | Sets the HTML `name` attribute for the `<button>`. Not valid if `url` is set.    |
 | url        | string | false    | If set, will create an HTML anchor link with the required classes and attributes |
 | attributes | object | false    | HTML attributes (for example, data attributes) to add to the button              |
+
+## SubNavigation
+
+| Name                   | Type                      | Required | Description                                                                                                                    |
+| ---------------------- | ------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| id                     | string                    | true     | The HTML `id` of the `<nav>` element. Used for the `aria-controls` attribute for the menu button displayed on small viewports. |
+| ariaLabel              | string                    | false    | The `aria-label` attribute added to the `<nav>` element. Defaults to “Main menu”.                                              |
+| itemsList              | array`<Item>`             | true     | Settings for an array of [list items](#item) for each navigation link                                                          |
+| currentPath            | string or array`<string>` | true     | The path of the current active page. Multiple paths can be provided using an array to highlight nested navigation.             |
+| toggleNavigationButton | array`<ToggleButton>`     | true     | Settings for the navigation [menu toggle button](#togglebutton) displayed on small viewports                                   |
+| removeHorizontalSubNav | boolean                   | false    | Set to “true” to remove the sub navigation                                                                                     |
```

Comment:
These two aren't used in subNavigation can you check all the others are used too and the descriptions are correct

Reply:
I have removed the toggle button, but the remove horizontalSubNav is used